.ci/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {label "clx-host-21514||clx-host-21518"}
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'10'))
+        timeout(time: 3, unit: 'MINUTES')
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        SRC_ROOT = "${env.WORKSPACE}"
+        ARTEFACT_DIR = "${env.WORKSPACE}/artefacts"
+        NFS_WORKSPACE = "/.autodirect/mtrswgwork/_ml_devops_team/workspace/ml-nccl-rdma-sharp-plugins-pr/${env.BUILD_NUMBER}"
+        CUDA_HOME = "/hpc/local/oss/cuda10.0"
```

Comment:
best not to hard-code system specific path into environment section. Use "module load /path/to/cuda/module or module load cuda-module-name" inside appropriate script.

Reply:
Removed this code and added `module load dev/cuda10.0` inside the scripts.